invenio_rdm_records/records/dumpers.py

```diff
@@ -17,14 +17,23 @@
 from pytz import utc
 
 
+def _format_date(date):
+    """Format the given date into ISO format."""
+    arrow = Arrow.fromtimestamp(calendar.timegm(date), tzinfo=utc)
+    return arrow.date().isoformat()
+
+
 class EDTFDumperExt(ElasticsearchDumperExt):
     """Elasticsearch dumper extension for EDTF dates support.
 
-    It adds two fields in the root of the document: publication_date_start
-    and publication_date_end, which correspond to the lower strict and upper
-    strict bounds. This are required for sorting. In an ascending sort (most
-    recent last), sort by publication_date_start to get a natural sort order.
-    In a descending sort (most recent first), sort by publication_date_end.
+    It parses the value (i.e. the EDTF level 0 string) of the specified
+    field in the document and adds a dictionary holding the values
+    :code:`{"lte": lower_strict, "gte": upper_strict}`, whose values correspond
+    to the lower strict and upper strict bounds.
+    They are required for sorting.
+    In an ascending sort (most recent last), sort by publication_date_start
+    to get a natural sort order. In a descending sort (most recent first),
+    sort by publication_date_end.
```

Comment:
moderate: This is not the case anymore, now we need to sort by `publication_date_range.gte` or `lte` right? I think it will still work, just needs updating this docstring.